lib/tasks/data_maintenance.rake

```diff
@@ -46,4 +46,9 @@ namespace :data_maintenance do
   task fix_move_agreed_for_non_prison_transfers: :environment do
     Move.where(move_agreed: false).where.not(move_type: 'prison_transfer').update_all(move_agreed: nil)
   end
+
+  desc 'fix nil estate for all existing production allocations data'
+  task fix_nil_allocations_estate: :environment do
+    Allocation.where(estate: nil).update_all(estate: :adult_male)
```

Comment:
if allocation.estate is allowed to be nil, why are all existing nil values being updated to `:adult_male`?

Reply:
Good question. This API endpoint is already in use and the PR extends to add support for an extra `estate` attribute. We can't make it mandatory yet as that would break the existing front end. So the intent is to deploy the code changes, then make a front end change to populate the `estate` attribute when creating an allocation. Once that's done and deployed the rake task can be run to fix the existing data. This approach avoids a breaking change to the API.

 We may also need a follow up PR to make `estate` mandatory if the suppliers are using the same endpoint - if so that's a breaking change so will have to be done under v2.